shell/linters.sh

```diff
@@ -17,23 +17,30 @@ if [[ -n $SKIP_LINTERS ]] || [[ -n $SKIP_VALIDATE ]]; then
   exit 0
 fi
 
+# add extra, per project, linters
+linters=("$DIR/linters"/*.sh)
+if [[ -z $workspaceFolder ]]; then
+  workspaceFolder="$(get_repo_directory)"
+fi
+if [[ -d "$workspaceFolder"/scripts/linters ]]; then
+  linters+=("${workspaceFolder}"/scripts/linters/*.sh)
```

Comment:
Nit: I don't think the curly braces are necessary.
```suggestion
  linters+=("$workspaceFolder"/scripts/linters/*.sh)
```

Reply:
It's personal styl pref of mine to always quote the entire string unless needed. So, I would also say this should be 

```suggestion
  linters+=("$workspaceFolder"/scripts/linters/"*.sh)
```